dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/util/KafkaUtils.java

```diff
@@ -1,37 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2024 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License 2.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *******************************************************************************/
-package com.ibm.ws.microprofile.reactive.messaging.fat.kafka.util;
-
-import java.util.Arrays;
-import java.util.List;
-import componenttest.topology.impl.LibertyServer;
-
-public class KafkaUtils {
-
-    private final static String KAFKA_REGEX = "E Error.*kafka";
```

Comment:
I'm surprised that removing this class didn't cause more changes to imports in other classes. Was it not referenced anywhere?

Reply:
The original PR, only added it, it was not added to any tests at the time, the use in the KafkaValidation test is the first time it is actually used